chart/kubeapps/templates/apprepository-deployment.yaml

```diff
@@ -52,7 +52,11 @@ spec:
            {{- end }}
             - --repo-sync-cmd=/asset-syncer
             - --namespace={{ .Release.Namespace }}
+            {{- if .Values.postgresql.existingSecret }}
             - --database-secret-name={{ .Values.postgresql.existingSecret }}
+            {{- else }}
+            - --database-secret-name={{ template "kubeapps.postgresql.fullname" . }}q
```

Comment:
is this `q` at the end something intented?

Reply:
mmh, no, it should've slipped after I tested this.